src/css/buttons.css

```
@@ -288,4 +267,167 @@
   color: var(--color-moon-200);
 }
 
+/* Iconlabel button */
+
+.a-btn--iconlabel .a-icon {
```

If this expression is composed then it's ok.

Otherwise, BEM supports something like:

```suggestion
.a-btn--icon-label .a-icon {
```

---

If we're going to keep "iconlabel" with no space, I think we should keep that...